cmd/aad-cli/cli/testdata/golden/TestUserGet/get_gecos

```diff
@@ -0,0 +1 @@
+
```

Comment:
hem, it’s empty in the test case, it probably should be set to compare it to something valuable :)